### DISCUSSION TITLE
Future: Add support for ipf images

### DISCUSSION THREAD

**Discussion by bnice** [dirkwhoffmann/vAmiga#494] [Ideas]
Hi,

Title says it all really add support for ipf and zipped ipf images.

Ben.

**Comment by toralux** [comment]
For MacOS library that reads ipf images, please checkout 
http://www.softpres.org/_media/files:ipfdevlib_mac-osx.dmg.zip?id=download&cache=cache

More information and developer reference documentation could be found here:
http://www.softpres.org/download

**Comment by dirkwhoffmann** [comment]
I am still unsure if it makes sense to add support to the emulator. Two reasons are holding me back at the moment:

- I haven't found a program that exists as IPF, but not as ADF.
- The framework does not exist in source code. Due to the age of the available version, I assume it is not compatible with ARM Macs.
